.github/dependabot.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+updates:
+    - package-ecosystem: "npm"
+      directory: "/"
+      schedule:
+          interval: "daily"
+      allow:
+          - dependency-type: "all"
+      target-branch: "develop"
```

Comment:
I think the default branch of the repo is develop so I don't think we need to include a `target-branch` here. I don't think it will do any harm for this to be included however.